### PR TITLE
jira job selection

### PR DESCRIPTION
**Pull request by psantos1113**

### CODE REVIEW

**Comment by bamandel** [comment]
In the future I believe we could modify the table select to not be field model specific and only allow a single select. That could take some time though, so this looks good!

File: ui/src/main/js/page/channel/jira/server/JiraServerDistributionConfiguration.js [hunk 1]
```diff
@@ -6,16 +6,50 @@ import CheckboxInput from 'common/component/input/CheckboxInput';
 import TextInput from 'common/component/input/TextInput';
 import CollapsiblePane from 'common/component/CollapsiblePane';
 import FieldMappingField from 'common/component/input/FieldMappingField';
+import { DISTRIBUTION_COMMON_FIELD_KEYS } from 'page/distribution/DistributionModel';
+import EndpointSelectField from '../../../../common/component/input/EndpointSelectField';
+import { createReadRequest } from '../../../../common/util/configurationRequestBuilder';
```
Comment (ChomickiM) [inline ui/src/main/js/page/channel/jira/server/JiraServerDistributionConfiguration.js:11]: We should use the absolute paths here

File: ui/src/main/js/page/channel/jira/server/JiraServerDistributionConfiguration.js [hunk 2]
```diff
@@ -6,16 +6,50 @@ import CheckboxInput from 'common/component/input/CheckboxInput';
...
 
 const JiraServerDistributionConfiguration = ({
-    data, setData, errors, readonly
+    csrfToken, data, setData, errors, readonly
 }) => {
+    const readRequest = () => {
+        const apiUrl = '/alert/api/configuration/jira_server?pageNumber=0&pageSize=25';
```
Comment (ChomickiM) [inline ui/src/main/js/page/channel/jira/server/JiraServerDistributionConfiguration.js:17]: I know this is probably very edge case, but would we need to state anywhere that we have an upper limit of 25 Jira Servers?

**Review by ChomickiM — APPROVED** [review]

File: ui/src/main/js/page/channel/jira/server/JiraServerDistributionConfiguration.js (hunk 1 above)
Comment (psantos1113) [reply ui/src/main/js/page/channel/jira/server/JiraServerDistributionConfiguration.js]: Thanks noticed that and fixed it.

File: ui/src/main/js/page/channel/jira/server/JiraServerDistributionConfiguration.js (hunk 2 above)
Comment (psantos1113) [reply ui/src/main/js/page/channel/jira/server/JiraServerDistributionConfiguration.js:17]: I added to the description we limit to the first 25 Jira servers

File: ui/src/main/js/common/component/input/GlobalConfigurationSelectInput.js [hunk 3]
```diff
@@ -0,0 +1,465 @@
...
+    const hasError = getValue().find((value) => value.missing);
+    return (
+        <ValueContainer {...props}>
+            {hasError && error}
+            {children}
+        </ValueContainer>
+    );
+};
+
+const GlobalConfigurationSelectInput = (props) => {
```
[the remark this answers is not in the document]
Comment (psantos1113) [reply ui/src/main/js/common/component/input/GlobalConfigurationSelectInput.js]: I removed it.  Maybe in the future we will create something like it.

**Pull request merged**